infra/base-images/base-builder/Dockerfile

```diff
@@ -55,8 +55,11 @@ RUN export PYTHON_DEPS="\
     rm -r /tmp/Python-$PYTHON_VERSION.tar.xz /tmp/Python-$PYTHON_VERSION && \
     apt-get remove -y $PYTHON_DEPS  # https://github.com/google/oss-fuzz/issues/3888
 
-# Install Atheris for Python fuzzing.
-RUN CLANG_BIN=/usr/local/bin/clang pip3 install atheris
+# Install latest atheris for python fuzzing.
+RUN pip3 install atheris
```

Comment:
Does this installation need to come later if we are using pyinstaller?

Reply:
Nope this package is for the fuzzer dependencies.